frontend/packages/ceph-storage-plugin/src/__tests__/independent-mode/details-card.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { ShallowWrapper, shallow } from 'enzyme';
+
+import { DashboardItemProps } from '@console/internal/components/dashboard/with-dashboard-resources';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+
+import { DashboardItems, detailsCardData } from '../../__mocks__/independent-mode-dashboards';
+import { DetailsCard } from '../../components/independent-dashboard-page/details-card/card';
+
+describe('DetailsCard', () => {
+  let wrapper: ShallowWrapper<DashboardItemProps>;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <DetailsCard
+        watchPrometheus={DashboardItems.watchPrometheus as any}
+        stopWatchPrometheusQuery={DashboardItems.stopWatchPrometheusQuery as any}
+        prometheusResults={DashboardItems.prometheusResults as any}
+        watchURL={DashboardItems.watchURL as any}
+        stopWatchURL={DashboardItems.stopWatchURL as any}
+        watchAlerts={DashboardItems.watchAlerts as any}
+        stopWatchAlerts={DashboardItems.stopWatchAlerts as any}
+        urlResults={DashboardItems.urlResults as any}
+        notificationAlerts={DashboardItems.notificationAlerts as any}
+        watchK8sResource={DashboardItems.watchK8sResource as any}
+        stopWatchK8sResource={DashboardItems.stopWatchK8sResource as any}
+        resources={detailsCardData.resources as any}
+      />,
+    ).dive();
+  });
+
+  it('Should render Card Header', () => {
+    expect(wrapper.find(DashboardCardHeader).exists()).toBe(true);
+  });
+
+  it('Should render Card Title', () => {
+    expect(wrapper.find(DashboardCardTitle).exists()).toBe(true);
+  });
+
+  it('Should render details properly', () => {
+    expect(wrapper.find('[data-test-id="cluster-name"]').text()).toEqual('foo');
+    expect(wrapper.find('[data-test-id="cluster-subscription"]').text()).toEqual('fooVersion');
+  });
+});
```

Comment:
This again should be part of details card. We can add these tests in console-shared.